app/posts/screening-invite/2025/09/2025-08-13-invite-breast-screening-pilot-user-research.md

```diff
@@ -0,0 +1,117 @@
+
+## What this user research is missing
+
+### Significant quantitative data 
+We can report that it was mainly SMS messages that participants saw, while just under a third of the sample read their NHS App messages and a few people received letters. You can see how each type of message was sent in sequence in [our earlier post on this.](https://screening-de-breast-scr-rui5fn.herokuapp.com/screening-invite/2025/06/invite-pilot-overview/#proposed-changes-to-how-people-are-invited-to-screening)
+
+However, the sample size of 150 isn’t large or varied enough to give us high levels of confidence for whether these numbers in the pilot would be the same if we rolled out for the whole breast screening population of around 6 million people. This research tells us *why* certain things might happen, not how often they might happen.
+
+### An understanding of how this would impact underserved groups
+The sample for this pilot deliberately chose people with the least barriers to accessing their screening appointment. A high priority for any future research would be to learn how digital first messaging would impact underserved groups who might struggle to access it the most, such as people with lower levels of digital capability or access. 
+
+## Noticing and reading messages
+
+Key findings were: 
+
+1) **People noticed their messages because our fallback system found the places they tend to check** 
+This was clear as we asked participants if they received their invitation in a channel they usually check – the majority said yes.
+
+    This is of course the purpose of the fallback system, to try digital messages first and use other channels when digital messages aren’t accessed.
+
+    In this sample, concerns around not having read receipts for SMS messages were mitigated by participants being experienced in receiving SMS messages for healthcare communications.
+
+2) **Messages can be noticed even when sent via a new channel**
+Even when participants received a message in a channel they don’t normally use, most of them still noticed it and attended their appointment.
+
+    This is something to keep in mind as we roll out and potentially hear more concerns from people who are used to receiving letters. While people’s attitude may be one of concern, their behaviour can still mean they access messages and attend appointments.
+
+3) **Messages can be effective even when people’s perception or memory of them is inaccurate**
+Many of the participants in this sample did not accurately recall the type of invitation they received. Some claimed they received an email, when we didn’t send any.
+
+    This is because it can be hard to keep track of which healthcare message is for which appointment, or distinguish between different types of notification on your phone. Also people were sent SMS reminders, which can be confused with the original invitation​.
```

Comment:
```suggestion
This is because it can be hard to keep track of which healthcare message is for which appointment, or distinguish between different types of notification on your phone. Also people were sent SMS reminders, which can be confused with the original invitation​.
```